.github/workflows/ci.yml

```diff
@@ -67,13 +67,13 @@ jobs:
         name: Set up JDK ${{ matrix.test-java-version }}-${{ matrix.vm }} for running tests
         uses: actions/setup-java@v2
         with:
-          distribution: adopt-${{ matrix.vm }}
+          distribution: temurin-${{ matrix.vm }}
```

Comment:
Looks like it fails on this part:

```
Error: No supported distribution was found for input temurin-hotspot
```

We should probably keep using `adopt-openj9` for openj9-based jvms - I don't think the `setup-java` action supports anything else: https://github.com/actions/setup-java#supported-distributions

Reply:
hey @sullis! let us know if you can get to fixing the build, otherwise one of us can do that

Reply:
@trask I don't have time to fix it. Can you finish it?

Reply:
I can take it over